src/client/Microsoft.Identity.Client/Platforms/Features/OpenTelemetry/OtelInstrumentation.cs

```diff
@@ -192,27 +197,30 @@ public void IncrementSuccessCounter(string platform,
                         new(TelemetryConstants.TokenSource, tokenSource),
                         new(TelemetryConstants.CacheRefreshReason, cacheRefreshReason),
                         new(TelemetryConstants.CacheLevel, cacheLevel));
+                //Add token Type to new counter?
```

Comment:
@neha-bhargava you mentioned that once a counter has 8 parameters it is full and that we may need to make a new one. The success counter has 8 so will this need be something like s_successCounter2("MsalSuccess2")? Or "MsalSuccess_Extra"?

Reply:
We discussed about combining the caller version sdk and id into one. And then adding the token type to this counter. So combining line 195 and 196 to something like `new(TelemetryConstants.CallerSdkId, callerSdkId ?? string.Empty + "," + callerSdkVersion ?? string.Empty)`